pkg/generate/config/config.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 // specify the overriding values for API operation parameters
 type OperationConfig struct {
 	OverrideValues map[string]string `json:"override_values"`
+	// CustomSetOutputOperation provides custom operation to process operation's output shape.
+	CustomSetOutputOperation string `json:"custom_set_output_operation,omitempty"`
```

Comment:
Probably just call this `SetOutputCustomMethod` or `SetOutputCustomMethodName` and change the description to say:

```
// SetOutputCustomMethod provides the name of the custom method on the
// `resourceManager` struct that will set fields on a `resource` struct
// depending on the output of the operation.
```

Reply:
Renamed to `SetOutputCustomMethodName` and updated the code comment per the suggestion.